test/main_test.go

```diff
@@ -102,6 +102,27 @@ var _ = Describe("go-testcov", func() {
 			})
 		})
 
+		It("fails when coverage is not ok and skipped generated files", func() {
+			withFakeGo("echo header > coverage.out; echo foo:1.2,1.3 0 >> coverage.out; echo generated.go:1.21.3 0 >> coverage.out" , func() {
+				writeFile("foo", "")
+				writeFile("generated.go", "")
+				expectCommand(
+					runGoTestWithCoverage,
+					[]interface{}{1, "", "foo new untested sections introduced (1 current vs 0 configured)\nfoo:1.2,1.3\n"},
+				)
+			})
+		})
```

Comment:
afaik we don't need this test since it basically tests the same, but better to have then not have :D